dts/bindings/power/state.yaml

```diff
@@ -28,4 +28,9 @@ properties:
         description: |
             Minimum residency duration in microseconds. It is the minimum time for a
             given idle state to be worthwhile energywise. It includes the time to enter
-            in this state.
\ No newline at end of file
+            in this state.
+    exit-latency-us:
+        type: int
+        required: false
+        description: |
+            Maximum time, in microseconds, required to wake up the CPU in this given state.
```

Comment:
same, Linux:
"          Worst case latency in microseconds required to exit the idle state.
          The exit-latency-us duration may be guaranteed only after
          entry-latency-us has passed."

Reply:
The last paragraph I have not included, we don't use/have entry latency.